### ISSUE TITLE
In gaussian_mesh_model.py line 149, why do "rotation.transpose(-2, -1)"

### ISSUE THREAD

**Issue by puhuajiang** [waczjoan/gaussian-mesh-splatting#29]
"line 147 rotation = torch.stack((v0, v1, v2), dim=1).unsqueeze(dim=1)
 line 148 rotation = rotation.broadcast_to((*self.alpha.shape[:2], 3, 3)).flatten(start_dim=0, end_dim=1)
 line 149 rotation = rotation.transpose(-2, -1)
"
In line 147, the rotation seems already good.
Thank you for your work!
